flags/flags.go

```diff
@@ -0,0 +1,52 @@
+package flags
```

Comment:
This package also ties us to `pflag`. Imo also a contract/abstraction with an impl for pflag should be used. Can we change this?

Reply:
I am not sure, whether contract/abstraction is a right place for the flags package

Reply:
i think we could put both cobra and pflag in a cmd package (together with main), but for now it's fine, let's talk next week. I would've loved to talk about that beforehand instead of just adding it without discussion, though. Nobody asked for a CLI, yet ;)